src/components/dumb/OeModalConfirmCancelChanges.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <vl-modal id="confirm-cancel-changes-modal">
+    <h2 class="vl-title vl-title--h2 vl-modal-dialog__title">
+      Ben je zeker dat je deze wijzigingen niet wilt bewaren?
+    </h2>
+    <div class="vl-u-spacer-bottom">
+      <p class="vl-u-spacer-bottom">
+        Bij het sluiten van deze fiche zullen je onbewaarde wijzigingen verloren gaan. Zorg ervoor dat je eerst je
+        wijzigingen bewaart, alvorens deze pagina te verlaten.
```

Comment:
> alvorens deze pagina te verlaten
miss beter 'alvorens deze tab te sluiten' ofzo?

Reply:
Deze component kan ook hergebruikt worden bij het cancelen van edit mode, in dat geval sluit de tab niet, dus is het niet beter van het algemeen te houden?